pylint_odoo/checkers/modules_odoo.py

```diff
@@ -114,6 +128,15 @@
 DFLT_EXTFILES_CONVERT = ['csv', 'sql', 'xml', 'yml']
 DFLT_EXTFILES_TO_LINT = DFLT_EXTFILES_CONVERT + [
     'po', 'js', 'mako', 'rst', 'md', 'markdown']
+DFLT_IMPORT_NAME_WHITELIST = [
+    # self-odoo
+    'odoo', 'openerp',
+    # Known external packages of odoo
+    'PIL', 'babel', 'dateutil', 'decorator', 'docutils', 'jinja2', 'ldap',
```

Comment:
That was my first option
- But the `pip package` is different to `import package`

But I have extracted the packages running this check from odoo 8.0, 9.0 and master:
`PYTHONPATH=~/build/OCA/pylint-odoo pylint --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" -d all -e missing-import-error --load-plugins=pylint_odoo odoo/addons/* odoo/openerp/addons/* odoo odoo/openerp -rn`